questionnaires/superset/README.md

```diff
@@ -112,3 +112,46 @@ docker-compose up
 ```
 docker-compose up
 ```
+
+## Share Dashboard
+
+### How to Share Dashboard Publicly
+
+Anyone with the link will be able to view the dashboard.
+
+- Click "Settings" menu on upper right corner
+- Click on "List Roles" menu item
+- Click on "+" icon to add a new role
```

Comment:
- Click on "+" icon to add a new role and name it clearly, for example "Public Access Role for Survey A in dataset PUBLIC_survey_a"